Action rightArmAction
Action leftArmAction
---
bool success